test/integration/activate_int_test.go

```diff
@@ -352,6 +352,36 @@ func (suite *ActivateIntegrationTestSuite) TestActivate_Command() {
 	cp.ExpectExitCode(0)
 }
 
+func (suite *ActivateIntegrationTestSuite) TestActivate_Default() {
```

Comment:
Rather than running a whole new activation test which slows down our overall test progress let's just add `--default` to one of our existing `state activate` tests. This is just one additional assertion.